strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java

```diff
@@ -177,6 +179,10 @@ public RepositoryPath resolveSibling(Path other)
 
     protected Path unwrap(Path other)
```

Comment:
I'm sorry... I think I've asked this before, but what was this unwrapping for?

Reply:
with this method you are able to safe get original `Path` instance from `RepositoryPath`.